### ISSUE TITLE
[<application>] need to edit my php file

### ISSUE THREAD

**Issue by gearcs** [bitnami/vms#776]
### Platform

Google Cloud Platform

### bndiagnostic ID [know more about bndiagnostic ID](https://docs.bitnami.com/general/how-to/understand-bndiagnostic/)

na

### bndiagnostic was not useful. Could you please tell us why?

not relevant to my issue

### Describe your issue as much as you can

heelo,
i have installed bitnami wp on my gcp and need to improve loading speed.

you advised earlier:

"htaccess files are disabled by default in the Bitnami solutions due to security and performance reasons. You can learn more about that here

https://docs.bitnami.com/general/apps/wordpress/administration/use-htaccess/

"

I checked this directory however i'm not sure which file has to be modified.

Basically, I need wp-config.php file to be editable as it has only read permission.

Can you please specify with step-by-step guide which directory and file has to be edited in order to get this php file to be editable?

![this one](https://user-images.githubusercontent.com/126983908/226208140-e61e356c-b87b-41bb-82eb-87e7a11ead13.png)

<img width="637" alt="sc 2023-03-17 at 11 34 56 AM" src="https://user-images.githubusercontent.com/126983908/226208132-ba1d95d5-3e6c-48dd-b288-fc369dc01b7a.png">

**Comment by jotamartos** [comment]
Please do not open multiple tickets for the same issue. Let's continue the conversations in the original thread 

https://github.com/bitnami/vms/issues/737
